api/python/quilt3/data_transfer.py

```diff
@@ -214,6 +216,15 @@ def check_head_object_works_for_client(s3_client, params):
 UPLOAD_ETAG_OPTIMIZATION_THRESHOLD = 1024
 
 
+def get_chunk_ranges(file_size):
+    adjuster = ChunksizeAdjuster()
+    chunk_size = adjuster.adjust_chunksize(s3_transfer_config.multipart_chunksize, file_size)
+    return [
+        (start, min(start + chunk_size, file_size) - 1)
```

Comment:
Very minor, but I think `(start, min(start + chunk_size, file_size))` would be better: pretty much everything in Python - ranges, lengths, etc. include the start, but not the end.

This will also answer @kevinemoore's question about `end - start + 1`.